core/schema/directory.go

```diff
@@ -55,7 +55,9 @@ func (s *directorySchema) Install() {
 			ArgDoc("permissions", `Permission given to the copied file (e.g., 0600).`),
 		dagql.Func("withoutFile", s.withoutFile).
 			Doc(`Retrieves this directory with the file at the given path removed.`).
-			ArgDoc("path", `Location of the file to remove (e.g., "/file.txt").`),
+			ArgDoc("path", `Location of the file to remove (e.g., "/file.txt").`).
+			ArgDoc("allowWildCard", `Allow wildcards in the path (e.g., "*.txt").`).
+			ArgDoc("allowNotFound", `Allow the operation to not fail if the directory does not exist.`),
```

Comment:
FYI - we would also want this on `container`. Side note (and not sure how to resolve this), it feels weird that we have so much repetition with the 4 methods there.